assembl/static2/js/app/pages/brightMirrorFiction.jsx

```diff
@@ -0,0 +1,68 @@
+// @flow
+import React, { Component, Fragment } from 'react';
+import { Grid, Row, Col } from 'react-bootstrap';
+import { I18n } from 'react-redux-i18n';
+// Graphql imports
+import { compose, graphql } from 'react-apollo';
+import BrightMirrorFictionQuery from '../graphql/BrightMirrorFictionQuery.graphql';
+// HOC imports
+import withLoadingIndicator from '../components/common/withLoadingIndicator';
+// Components imports
+import FictionHeader from '../components/debate/brightMirror/fictionHeader';
+import FictionBody from '../components/debate/brightMirror/fictionBody';
+// Type imports
+import type { CircleAvatarType } from '../components/debate/brightMirror/circleAvatar';
+import type { FictionHeaderType } from '../components/debate/brightMirror/fictionHeader';
+import type { FictionBodyType } from '../components/debate/brightMirror/fictionBody';
+
+type BrightMirrorFictionType = {
+  // contentLocale: string,
+  data: {
+    fiction: BrightMirrorFictionFragment
+  }
+};
+
+class BrightMirrorFiction extends Component<BrightMirrorFictionType> {
+  render() {
+    // const { contentLocale } = this.props;
+    const { fiction } = this.props.data;
+
+    // Define components props
+    const circleAvatarProps: CircleAvatarType = {
```

Comment:
Is it useful to redo a check for flow here as flow manage the checks when we pass the props to the component already?

Reply:
Yes, why not pass directly the props to the component instead of using a variable?